.github/workflows/build.yaml

```
@@ -52,6 +52,15 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  docs-build:
+    if: ${{ startsWith(github.ref, 'refs/heads/branch-') }}
+    needs: python-build
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/custom-job.yaml@cuda-118
+    with:
+      build_type: branch
+      arch: "amd64"
+      run_script: "ci/build_docs.sh"
```

```suggestion
    with:
      build_type: pull-request
      node_type: "gpu-latest-1"
      arch: "amd64"
      container_image: "rapidsai/ci:latest"
      run_script: "ci/build_docs.sh"
```

This `with` block should match the one in `pr.yaml`